dallinger/deployment.py

```diff
@@ -25,15 +26,27 @@
 from dallinger.utils import GitClient
 
 
+config = get_config()
+OSX_CHROME_PATH = '/Appliations/Google Chrome.app/MacOS/Google Chrome'
```

Comment:
```suggestion
OSX_CHROME_PATH = '/Applications/Google Chrome.app/MacOS/Google Chrome'
```

Reply:
Thanks!

Reply:
I notice on my system, the app is at:
```
/Applications/Google\ Chrome.app/Contents/MacOS/Google Chrome
```
I wonder if this should be dealt with via `Configuration`?